test/extended/image_ecosystem/s2i_python.go

```diff
@@ -20,15 +20,15 @@ var _ = g.Describe("[image_ecosystem][python][Slow] hot deploy for openshift pyt
 	defer g.GinkgoRecover()
 
 	var (
-		oc               = exutil.NewCLI("s2i-python", exutil.KubeConfigPath())
-		djangoRepository = "https://github.com/sclorg/django-ex.git"
-		modifyCommand    = []string{"sed", "-ie", `s/'count': PageView.objects.count()/'count': 1337/`, "welcome/views.py"}
-		pageCountFn      = func(count int) string { return fmt.Sprintf("Page views: %d", count) }
-		dcName           = "django-ex"
-		rcNameOne        = fmt.Sprintf("%s-1", dcName)
-		rcNameTwo        = fmt.Sprintf("%s-2", dcName)
-		dcLabelOne       = exutil.ParseLabelsOrDie(fmt.Sprintf("deployment=%s", rcNameOne))
-		dcLabelTwo       = exutil.ParseLabelsOrDie(fmt.Sprintf("deployment=%s", rcNameTwo))
+		oc             = exutil.NewCLI("s2i-python", exutil.KubeConfigPath())
+		djangoTemplate = "django-psql-example"
```

Comment:
This seems unrelated to cakephp - was this another 4.x fix that we're backporting?

Reply:
yes part of the image-eco fixes that are in the commit message but not the PR title ... will adjust PR title

Reply:
After this merged, these tests started permanently failing:

![image](https://user-images.githubusercontent.com/429763/141981338-98a2d85b-1859-4c38-9b77-dd5caa2ce61d.png)

Reply:
the PR is unrelated ... looking at your graph above, failures started happening Nov 2021

this PR merged Nov 2020

I'll move the analysis of things over to https://bugzilla.redhat.com/show_bug.cgi?id=2023238 and the recent failures noted there

Reply:
Yea sorry about that, I had searched origin for `ecosystem python` and saw 11/9.  Thanks for looking at it, the cause for the failures must be elsewhere